services/api-server/src/simcore_service_api_server/services/rabbitmq.py

```diff
@@ -1,8 +1,15 @@
 import logging
 
 from fastapi import FastAPI
+from prometheus_fastapi_instrumentator import Instrumentator
+from servicelib.fastapi.prometheus_instrumentation import (
+    setup_prometheus_instrumentation,
+)
+from servicelib.fastapi.rabbitmq import get_rabbitmq_client
 from servicelib.rabbitmq import RabbitMQClient, wait_till_rabbitmq_responsive
 from settings_library.rabbit import RabbitSettings
+from simcore_service_api_server.api.dependencies.rabbitmq import get_log_distributor
```

Comment:
please use relative imports, that will also fix the difference between Ruff and iSort

Reply:
Fixed, thanks for reminding me